lib/node_modules/@stdlib/math/stats/lib/index.js

```diff
@@ -63,6 +63,15 @@ setReadOnly( ns, 'chi2gof', require( '@stdlib/math/stats/chi2gof' ) );
 */
 setReadOnly( ns, 'incr', require( '@stdlib/math/stats/incr' ) );
 
+/**
```

Comment:
This is not something that you should include in this PR. This is something that happens after a PR is merged and functionality added to the library.